### PR TITLE
Update example command in README.md

### PR DESCRIPTION
**Pull request by EPashkin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 Running:
 ```shell
-cargo run --release -- -d ../gir-files Gtk-3.0
+cargo run --release -- -c Gir_Gtk.toml -d ../gir-files -o ./gtk
```
Comment (gkoz) [inline README.md]: Just `gtk` might be nicer looking than `./gtk`, also maybe just call it something neutral like `out_dir`?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
...
 ```
 where `gir-files` contains the [GIR definitions](https://github.com/gkoz/gir-files).
+In `./gtk` will be placed generated files.
```
Comment (gkoz) [inline README.md]: This would be "The generated files will be placed in ..."

**Comment by gkoz** [comment]
Thanks for the improvements. There're some nits but it's a good direction overall.

**Comment by EPashkin** [comment]
`out_dir` to general and and it ask for `-c name.toml` so it will be not command that just copy-paste-run

**Comment by gkoz** [comment]
Thanks.
@homu r+

**Pull request merged**